backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
I'm all for it! I think just dumping `then` is a fair compromise between spec compliance and usability. Our `this`es are broken these days anyway because of arrow functions.
Your call!